src/common/maps/log-level-to-console-color.map.js

```diff
@@ -0,0 +1,9 @@
+import { LogLevel } from "../enums/enums.js";
+
+const LogLevelToConsoleColor = {
+  [LogLevel.LOG]: "\x1b[32m",
+  [LogLevel.WARNING]: "\x1b[33m",
+  RESET: "\x1b[0m",
```

Comment:
```suggestion
  reset: "\x1b[0m",
```
since a `logLevelToConsoleColor` key`[LogLevel.LOG]` is `LogLevelToConsoleColor.log`